app/code/Magento/Store/Model/Store.php

```diff
@@ -1344,16 +1367,20 @@ public function getIdentities()
     }
 
     /**
+     * Return Store Path
+     *
      * @return string
      */
     public function getStorePath()
     {
         $parsedUrl = parse_url($this->getBaseUrl());
-        return isset($parsedUrl['path']) ? $parsedUrl['path'] : '/';
+        return $parsedUrl['path'] ?? '/';
     }
 
     /**
-     * {@inheritdoc}
```

Comment:
Please, revert changes related to @inheritdoc. Replace {@inheritdoc} with @inheritdoc